frontend/components/course-listings/listings.tsx

```diff
@@ -0,0 +1,109 @@
+import Head from 'next/head';
+import Image from 'next/image';
+import Box from '@mui/material/Box';
+import Grid from '@mui/material/Grid';
+import Card from '@mui/material/Card';
+import Container from '@mui/material/Container';
+import InputLabel from '@mui/material/InputLabel';
+import MenuItem from '@mui/material/MenuItem';
+import FormControl from '@mui/material/FormControl';
+import Select, { SelectChangeEvent } from '@mui/material/Select';
+import CardContent from '@mui/material/CardContent';
+import CardHeader from '@mui/material/CardHeader';
+import Tabs from '@mui/material/Tabs';
+import Tab from '@mui/material/Tab';
+import Typography from '@mui/material/Typography';
+import Button from '@mui/material/Button';
+import { CourseEvaluationListEntry } from 'utils/api';
+import { Reviews } from '@mui/icons-material';
+import EvaluationList from './evaluationList';
+
+//for now reviews as string but will need to change in reviewer list branch (similar to evaluations)
+type Props = {
+  evaluations: CourseEvaluationListEntry[];
+  reviews: String[];
+};
+
+interface TabPanelProps {
+  children?: React.ReactNode;
+  index: number;
+  value: number;
+}
+
+function TabPanel(props: TabPanelProps) {
+  const { children, value, index, ...other } = props;
+
+  return (
+    <div
+      role="tabpanel"
+      hidden={value !== index}
+      id={`simple-tabpanel-${index}`}
+      aria-labelledby={`simple-tab-${index}`}
+      {...other}
+    >
+      {value === index && (
+        <Box sx={{ p: 3 }}>
+          <Typography>{children}</Typography>
+        </Box>
+      )}
+    </div>
+  );
+}
+
+function a11yProps(index: number) {
+  return {
+    id: `simple-tab-${index}`,
+    'aria-controls': `simple-tabpanel-${index}`,
+  };
+}
+
+function Listings({ evaluations, reviews }: Props) {
+  const [value, setValue] = React.useState(0);
+
+  const handleChangeTab = (event: React.SyntheticEvent, newValue: number) => {
+    setValue(newValue);
+  };
+
+  return (
+    <Container
+      sx={{
+        display: 'flex',
+        flexDirection: 'column',
+        justifyContent: 'center',
+        alignItems: 'center',
+      }}
+    >
+      <Box
+        sx={{ borderBottom: 1, borderColor: 'divider', backgroundColor: '#0E91AC', width: '90%' }}
+      >
+        {evaluations.length > 0 && reviews.length > 0 ? (
+          <Tabs value={value} onChange={handleChangeTab} variant="fullWidth">
+            <Tab label="COURSE EVALUATIONS" {...a11yProps(0)} />
+            <Tab label="REVIEW COURSES" {...a11yProps(1)} />
+          </Tabs>
+        ) : evaluations.length > 0 ? (
+          <Tabs value={value} onChange={handleChangeTab} variant="fullWidth">
+            <Tab label="COURSE EVALUATIONS" {...a11yProps(0)} />{' '}
+          </Tabs>
+        ) : (
+          <Tabs value={value} onChange={handleChangeTab} variant="fullWidth">
+            <Tab label="Reviews" {...a11yProps(0)} />
+          </Tabs>
+        )}
+      </Box>
```

Comment:
this tab was extremely hard to read , maybe just move it inside the main box you've already created?

Reply:
I have changed it so that it is easier to read. I had to use conditions so that the coordinator/reviewer tabs only show up if they are a coordinator or reviewer. But I changed it so it is much better than this...

Reply:
this was the code I was kind of referring to

```
 <Container
      sx={{
        display: 'flex',
        flexDirection: 'column',
        justifyContent: 'center',
        alignItems: 'center',
      }}
    >
      <Box sx={{ backgroundColor: '#EEEEEE', width: '100%' }}>
        <Tabs value={tabsValue} onChange={handleChangeTab} variant="fullWidth">
          {courseEvaluationListEntries.length > 0 && (
            <Tab label="COURSE EVALUATIONS" {...a11yProps(0)} />
          )}
          {reviews.length > 0 && <Tab label="REVIEW COURSES" {...a11yProps(1)} />}
        </Tabs>
        <TabPanel value={tabsValue} index={0}>
          <CoordinatorList list={courseEvaluationListEntries} />
        </TabPanel>
        <TabPanel value={tabsValue} index={1}>
          Item Two
        </TabPanel>
      </Box>
    </Container>
```

> this tab was extremely hard to read 

I was referring to the user interface, not the code...

Here's my recommendation
![Screen Shot 2022-07-23 at 10 24 07 pm](https://user-images.githubusercontent.com/44391389/180609277-6785f1a4-c6fe-4ab9-8d5c-50d10820d6ff.png)
<img width="1203" alt="Screen Shot 2022-07-23 at 10 24 04 pm" src="https://user-images.githubusercontent.com/44391389/180609279-cb42392e-fa54-44ec-ab43-3bca60de76c5.png">

Reply:
this is what it will look like if there's two tabs
<img width="1184" alt="Screen Shot 2022-07-23 at 10 26 59 pm" src="https://user-images.githubusercontent.com/44391389/180609382-836883f7-d712-476e-84d1-cfe4e96be7cf.png">

Reply:
Also just a general guideline for UI design... 

**Primary actions** should correspond to **primary colors**. In this UI design, "Create new evaluation" is a primary action similar to "View", so they should usually be the same color which is a "primary color".

"Show completed" will have to be a secondary color as it is a secondary action.